### ISSUE TITLE
Move incoming answers

### ISSUE THREAD

**Issue by drjakob** [tactilenews/100eyes#232]
As an editor, I want to be able to assign incoming answers to other questions with two clicks in order to have answers arriving too late for the right question.

**Comment by drjakob** [comment]
Pleas code this (20.6.2020)

**Comment by tillprochaska** [comment]
I think we discussed different UIs for this in the past. One of them was to drag and drop into the correct request. I’m not sure how exactly that interaction would work – probably we’d have to show a list of all request at the side of the screen when users start dragging a reply.

I could imagine that dragging a reply might actually be a bit difficult and would go for a simpler solution:

<img width="845" alt="Screen Shot 2020-10-16 at 15 50 24" src="https://user-images.githubusercontent.com/1512805/96266996-9a813680-0fc7-11eb-8b7c-53f95613f4a2.png">

* Clicking the "verschieben" button opens a model that allows you to select one of the previous requests.
* By default, the current request is selected.
* The list of requests ist sorted in inverse-chronological order, i.e. latest request is at the top.
* Only three clicks are needed to move a reply. 
* After moving a questions, editors stay on the same page and can continue browsing through the other replies.

**Comment by tillprochaska** [comment]
@FrauCsu @drjakob Would that be a good solution for your use case?

**Comment by FrauCsu** [comment]
YES, loving it, @tillprochaska

**Comment by tillprochaska** [comment]
@FrauCsu @drjakob: The list of questions you might want to move a reply to might get quite long for long-running projects.

Just to be clear, this is the use case:

* Editor sends request A on Monday.
* Editor sends request B on Tuesday morning.
* User replies to requests A and B on Tuesday evening.
* Both replies are attached to the request B, although the first reply actually belongs to request A.
* Editor sends request C on Wednesday.

I think we can derive the following constraints from this use case, but I’m not sure if those assumptions are correct:
* We only ever want to move replies to requests older than the request the reply is currently attached to. In the example above, we wouldn’t want to move a reply to request C. When the user replied on Tuesday, they didn’t yet receive request C, so their replies belong to request C.
* Most replies will belong to a recent request, i.e. there’s no need to show super old requests in the modal.

**Comment by drjakob** [comment]
Yes, constrains are correct. The use case is recent questions. I think we need more information, e.g. date of the question. I think 5 requests will be sufficient.

**Comment by tillprochaska** [comment]
Displaying the date of the requests is a good idea!

**Comment by FrauCsu** [comment]
Question solved or need further discussion, @tillprochaska ?

**Comment by tillprochaska** [comment]
Yes!

**Comment by tillprochaska** [comment]
@drjakob: This would be the current UI – any feedback?

<img width="675" alt="Screen Shot 2020-10-20 at 15 27 35" src="https://user-images.githubusercontent.com/1512805/96593054-610f3a80-12e9-11eb-9f12-bb12d91329f2.png">

**Comment by FrauCsu** [comment]
Yes, @tillprochaska ! Good solutions. Not more than five older questions displayed is also fine.
